moesifwsgi/middleware.py

```diff
@@ -103,21 +103,39 @@ def __call__(self, environ, start_response):
                         environ['REQUEST_METHOD'],
                         self.logger_helper.request_url(environ),
                         self.client_ip.get_client_address(environ),
-                        self.logger_helper.get_user_id(environ, self.settings, self.app, self.DEBUG),
-                        self.logger_helper.get_company_id(environ, self.settings, self.app, self.DEBUG),
-                        self.logger_helper.get_metadata(environ, self.settings, self.app, self.DEBUG),
-                        self.logger_helper.get_session_token(environ, self.settings, self.app, self.DEBUG),
                         [(k, v) for k,v in self.logger_helper.parse_request_headers(environ)],
                         *self.logger_helper.request_body(environ)
                     )
 
         def _start_response(status, response_headers, *args):
             # Capture status and response_headers for later processing
             data_holder.capture_response_status(status, response_headers)
+
+            if response_headers:
+                response_headers_dict = {}
+                try:
+                    for pair in response_headers:
+                        response_headers_dict[pair[0].lower()] = pair[1]
```

Comment:
any reason why converting the key to the lowercase?

Reply:
Since we the headers in falcon is saving the lowercase as keys, so I changed all to lower to keep the middleware safe. Let me know if you have better suggestion.

Reply:
Removed lower